.drone.yml

```diff
@@ -1,3 +1,29 @@
+---
+kind: pipeline
+type: kubernetes
+name: deploy tag
+trigger:
+  event:
+    - tag
+  branch:
+    - main
+
+steps:
+  - name: cs-dev
+    image: quay.io/ukhomeofficedigital/helm:v3.5.4
+    environment:
+      KUBE_CLUSTER_NAME: acp-notprod
+      KUBE_NAMESPACE: cs-dev
+      KUBE_SERVER: https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
+      KUBE_TOKEN:
+        from_secret: cs_notprod
+      VALUES_FILE: '--values ./helm/values-notprod.yaml'
```

Comment:
I still think we should pass in a values-namespace.yaml file, maybe in additoin to the not-prod.yaml? I wonder if it's possible not fail if it doesn't exist?

Reply:
it will apply them in order so notprod first, then *cs-*.yaml